Spigot-API-Patches/0243-Added-SoundBuilder.patch

```diff
@@ -0,0 +1,349 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fruxz <cedricspitzer@outlook.de>
+Date: Fri, 18 Dec 2020 01:02:39 +0100
+Subject: [PATCH] Added SoundBuilder
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/sound/SoundBuilder.java b/src/main/java/com/destroystokyo/paper/sound/SoundBuilder.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..803c51548d3ea74c6e6872908d751b42e0a420be
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/sound/SoundBuilder.java
+@@ -0,0 +1,337 @@
++package com.destroystokyo.paper.sound;
```

Comment:
Paper is moving to the `io.papermc` package

Reply:
yes please move the package

Reply:
Thank you for the positive feedback! I have tried to drag the class into the new Paper Package via the instructions. I hope everything worked as it should. If something went wrong, I would be very happy about support. :D